core/src/main/scala/org/apache/spark/executor/ExecutorExitCode.scala

```diff
@@ -45,6 +45,10 @@ object ExecutorExitCode {
    */
   val HEARTBEAT_FAILURE = 56
 
+  /** The default uncaught exception handler was reached and the exception was thrown by
+   * TaskReaper. */
+  val KILLED_BY_TASK_REAPER = 57
+
   def explainExitCode(exitCode: Int): String = {
```

Comment:
@bozhang2820, actually, whoops, I had overlooked this, but it looks like we should probably add a new case to `explainExitCode` here for the new `KILLED_BY_TASK_REAPER`

Reply:
Thanks! Added a new case in `explainExitCode`.